src/VideoPlayer.cs

```diff
@@ -230,6 +230,8 @@ void Update()
             if (ReadyState <= ReadyState.HaveNothing)
             {
                 currentVideoFrame = null;
+                renderTarget?.Dispose();

```

Comment:
The `renderTarget` should be set to null as well so we can be sure it's disposed only once.